src/gui_common/tooltip/ToolTipManager.cs

```diff
@@ -378,12 +378,14 @@ public void OnLanguageChanged()
         // Organelle tooltips
         foreach (var organelle in SimulationParameters.Instance.GetAllOrganelles())
         {
-            var tooltip = GetToolTip(organelle.InternalName, "organelleSelection");
+            var tooltip = GetToolTip(organelle.InternalName, "organelleSelection") as SelectionMenuToolTip;
```

Comment:
I think it would be good to do the cast in a second step and print a warning if the cast fails, as because written like this, this can silently fail as the tooltip doesn't exist and wrong type will now be both handled just by:

```
            if (tooltip == null)
                continue;
```

so a future bug may be hard to find.